core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
```

Comment:
Ok Sean, I understood your point. I will recreate pull request with more
precise changes and will also run "sbt scalastyle" for it.

Should i close this pull request??

On Tue, Apr 26, 2016 at 2:48 PM, Sean Owen <notifications@github.com> wrote:

> In core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala
> <https://github.com/apache/spark/pull/12697#discussion_r61053012>:
>
> > @@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
> >  import org.apache.spark.SecurityManager
> >  import org.apache.spark.metrics.MetricsSystem
> >
> > +import org.slf4j.Logger
> > +import org.slf4j.LoggerFactory
>
> The problem you describe is that log messages aren't plumbed through to a
> logger, right? that shouldn't require changing appender config for log4j.
> Your JIRA says "Slf4jsink.scala should have class name for log4j to print
> metrics in log files" but your code has a package name.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12697/files/5a437a12fb0c6cfdd7119d63bebc3f3f2c935d5f#r61053012>
>



-- 
Mihir Monani
(+91)-9429473434